src/main/java/emissary/parser/SessionProducer.java

```diff
@@ -29,12 +28,10 @@ public class SessionProducer {
      * Creates a SessionProducer
      *
      * @param sp The SessionParser, which has parsed theContent.
-     * @param myKey Value to stick into the transform history, obsolete
      * @param initialForms Forms to be preloaded onto the form stack.
      */
-    public SessionProducer(SessionParser sp, String myKey, List<String> initialForms) {
```

Comment:
We probably should deprecate this constructor first, as it will have downstream implications, but still move to the new constructor.

Reply:
good idea, you're right should probably do that. I'll implement it